spec/yaml/swiftnav/sbp/solution_meta.yaml

```diff
@@ -124,13 +124,12 @@ definitions:
  #############################################
  # for GNSS sensors
  - GNSSInputType:
-     id: 0xFFE7
-     short_desc: Flags for a given GNSS sensor used as input for the fuzed solution.
+     short_desc: Instruments the physical type of GNSS sensor input to the fuzed solution.
      public: true
+     description_only: true
```

Comment:
How about `embedded_type: true` or `generate_docs: false`?

Reply:
Thanks so much for the review Jason.  

Yes I agree now that renaming linux is bad.  I will drop those messages from PDF for now instead.

I like "embedded_type" a lot and will go with that instead.  I couldn't think of a good name so went with an obviously bad one hoping a better one would come to me.

The unit tests are wrong since they call a message a message that isn't, so I will update them.